client/src/main/java/org/apache/celeborn/client/read/CelebornInputStream.java

```diff
@@ -517,25 +511,37 @@ public int read(byte[] b, int off, int len) throws IOException {
     }
 
     @Override
-    public void close() {
-      int locationsCount = locations.length;
-      logger.debug(
-          "total location count {} read {} skip {}",
-          locationsCount,
-          locationsCount - skipCount.sum(),
-          skipCount.sum());
-      if (currentChunk != null) {
-        logger.debug("Release chunk {}", currentChunk);
-        currentChunk.release();
-        currentChunk = null;
-      }
-      if (currentReader != null) {
-        logger.debug("Closing reader");
-        currentReader.close();
-        currentReader = null;
-      }
-      if (containLocalRead) {
-        ShuffleClient.printReadStats(logger);
+    public synchronized void close() {
+      if (!closed) {
+        int locationsCount = locations.length;
+        logger.debug(
```

Comment:
It's recommended to log `appShuffleId`, `shuffleId` and `partitionId` for debug.

Reply:
done